test/scullery.ini

```diff
@@ -0,0 +1,25 @@
+[box]
+bootstrap=test/bootstrap-salt-roots.sh
+
+[box.tumbleweed]
+name=tumbleweed-salt.x86_64
+image=https://download.opensuse.org/repositories/home:/crameleon:/appliances/openSUSE_Tumbleweed/boxes/tumbleweed-salt.x86_64.json
+
+[box.leap]
+name=leap-salt.x86_64
+image=https://download.opensuse.org/repositories/home:/crameleon:/appliances:/Leap-15.4/images/boxes/leap-salt.x86_64.json
```

Comment:
Please upgrade to 15.5 ;-)

Reply:
SLES 15 SP5 is not released yet. :-) I can add a second box to keep both 15.4 and 15.5 though.

Reply:
Added and adjusted the sample grains test to use 15.5.